docs/configuration.md

```diff
@@ -629,3 +629,104 @@ window.$docsify = {
   topMargin: 90, // default: 0
 };
 ```
+
+## vueComponents
+
+- type: `Object`
+
+Registers Vue components using the component name as the key with an object containing Vue options as the value.
+
+```js
+window.$docsify = {
+  vueComponents: {
+    'button-counter': {
+      template: `
+        <button @click="count += 1">
+          You clicked me {{ count }} times
+        </button>
+      `,
+      data() {
+        return {
+          count: 0,
+        };
+      },
+    },
+  },
+};
+```
+
+```markdown
+<button-counter></button-counter>
+```
+
+<output data-lang="output">
+  <button-counter></button-counter>
+</output>
+
+## vueGlobalOptions
+
+- type: `Object`
+
+Specifies Vue options to be shared throughout your site. These options will be used when Docsify detects Vue content in the main content area that has not been previously mounted via [vueMountOptions](#vuemountoptions), [vueComponents](#vuecomponents), or a markdown `<script>`.
```

Comment:
Do these options apply to any of the components made with `vueMountOptions`?

Reply:
@trusktr --

No. That's what the following part of the description is trying to make clear: 

> These options will be used when Docsify detects Vue content in the main content area that has not been previously mounted via vueMountOptions, vueComponents, or a markdown `<script>`.

If that isn't clear, let me know and I can try to rework the description.

Reply:
@trusktr Update the description as follows:

> Specifies options for use with Vue content not explicitly mounted with `vueMounts`, `vueComponents`, or a markdown `<script>`. Changes to global `data` will persist and be reflected anywhere global references are used.

Hopefully that makes it more clear.